examples/vpc-example/README.md

```diff
@@ -0,0 +1,59 @@
+# AWS VPC
+
+A virtual private cloud (VPC) is a virtual network that closely
+resembles a traditional network that you'd operate in your own data
+cente. The AWS Puppet module allows for describing the various moving
+parts of VPC in the Puppet DSL. This can be useful for creating and
+sharing duplicate environments or alternatively as part of an audit
+mechanism to ensure your VPC configuration is as intended.
+
+## What
+
+For this examplei= we're going to create a VPC, a subnet, a route table
+and an internet gateway. This broadly follows the [getting started
+example](http://docs.aws.amazon.com/AmazonVPC/latest/GettingStartedGuide/Wizard.html)
+from the official Amazon documentation.
```

Comment:
I'll create a separate example for the VPN resources. The issue with those is that you can't really having a 'working' example without having the hardware at your end - we can just create some resources in AWS with dummy data.